lms/services/__init__.py

```diff
@@ -10,7 +10,6 @@
     LTILaunchVerificationError,
     LTIOAuthError,
     LTIOutcomesAPIError,
-    NoConsumerKey,
```

Comment:
This isn't something we retrieve any more, `oauthlib` does it for us. This means we never have an opportunity to raise this.